### ISSUE TITLE
Test if nested virtualization is enabled when using VirtualBox engine for the GNS3 VM

### ISSUE THREAD

**Issue by grossmj** [GNS3/gns3-server#1377]
VirtualBox is not recommended to run the GNS3 VM, let's consider hiding it, put a warning or drop the support altogether.

**Comment by ehlers** [comment]
Another issue of VirtualBox is, that it doesn't run in parallel to other virtualization products, see https://www.virtualbox.org/manual/ch10.html#hwvirt

Therefore I suggest to create a deprecated warning on 2.2 and remove it with 2.4 (or 3.0?).

**Comment by Thorbijoern** [comment]
why is VirtualBox not recommended?

**Comment by grossmj** [comment]
Because VirtualBox doesn't support nested virtualization, preventing to run Qemu VMs inside the GNS3 VM. Most vendor appliances (Cisco IOSv, Arista vEOS etc.) are meant to run with Qemu.

**Comment by grossmj** [comment]
> Therefore I suggest to create a deprecated warning on 2.2 and remove it with 2.4 (or 3.0?).

I have marked it as deprecated and we will consider removing the support altogether in a future release.

**Comment by grossmj** [comment]
This can be closed when nested virtualization is supported by VirtualBox for both AMD and Intel.

**Comment by grossmj** [comment]
Comment from Andrew on Slack.

> Is the Virtualbox GNS3 VM going to be updated so that it automatically enables nested virt,  once they get Intel support working?  My laptop has an AMD processor, and the GNS3 VM works great with it, but I did have to manually edit the VMs properties in Virtualbox to enable the nested virt selections after the fact  (after setting up GNS3 to use it, it still defaulted to KVM=false initially).   I guess worst case, a couple of screenshots on a support page could work for now.

**Comment by grossmj** [comment]
Fixed
